demo/demoproject/settings.py

```diff
@@ -64,14 +64,24 @@
 
 
 # BEGIN middlewares
-MIDDLEWARE_CLASSES = [
-    'django.middleware.common.CommonMiddleware',
-    'django.contrib.sessions.middleware.SessionMiddleware',
-    'django.middleware.csrf.CsrfViewMiddleware',
-    'django.contrib.auth.middleware.AuthenticationMiddleware',
-    'django.contrib.messages.middleware.MessageMiddleware',
-    'django_downloadview.SmartDownloadMiddleware'
-]
+if StrictVersion(get_version()) >= StrictVersion('1.10'):
+    MIDDLEWARE = [
```

Comment:
Instead of doing so, cannot we defined MIDDLEWARE_CLASSES  = MIDDLEWARE if Django < 1.10 ? It would avoid the duplication of middleware lists

Reply:
Fair enough. Seems when `MIDDLEWARE_CLASSES` and `MIDDLEWARE` are both defined in Django >=1.10 `MIDDLEWARE` wins. Adjusted code likewise to avoid duplication.